doc/GoogleLanguageConnector.md

```diff
@@ -0,0 +1,50 @@
+# reshuffle-google-language-connector
```

Comment:
used to be the package name so better to use the main title here (see Google Sheets doc) so it's consistent